tests/test_representation_manager.cc

```diff
@@ -48,7 +48,8 @@ namespace rascal {
     true_order << 0, 1, 3, 2, 4;
 
     auto test_order = internal::SortCoulomMatrix<
-        Option::CMSortRowNorm>::get_coulom_matrix_sorting_order(test_matrix);
+      internal::CMSortAlgorithm::RowNorm>::get_coulom_matrix_sorting_order(
```

Comment:
Maybe fix the spelling with search and replace?
```suggestion
      internal::CMSortAlgorithm::RowNorm>::get_coulomb_matrix_sorting_order(
```